ThunderCloud/SpotlightListItemCell.swift

```diff
@@ -275,9 +282,21 @@ open class SpotlightListItemCell: StormTableViewCell, ScrollOffsetManagable {
                 pageIndicator.isHidden = spotLights.count < 2
             }
             pageIndicator.numberOfPages = spotlights?.count ?? 0
+            redrawPageIndicatorImages()
             collectionView.reloadData()
         }
     }
+
+    private func redrawPageIndicatorImages() {
+        // Performance check
+        guard Self.preferredPageIndicatorImage != nil || Self.selectedPageIndicatorImage != nil else {
```

Comment:
Going to approve anyway, but wondering what would happen if one was `nil` and one was not